ci/10_linux_build.bash

```diff
@@ -7,11 +7,11 @@ source "$ci_dir/emsdk/emsdk_env.sh"
 cd "$ci_dir/.."
 
 export EMSCRIPTEN="$ci_dir/emsdk/upstream/emscripten"
-export PATH=$PATH:"$(pwd)/node_modules/.bin/"
+export PATH="$PATH:$(pwd)/node_modules/.bin/"
 
 rm -rf build && mkdir build
 cd build
 mkdir deps
 mkdir assets
 emcmake cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=OFF ..
-cmake --build . -- -j
+cmake --build . -- -j"$(nproc)"
```

Comment:
Without specifying the job limit, the build fails if the env. `CMAKE_GENERATOR` is set to `"Ninja"`.